sdk/mediaservices/arm-mediaservices/README.md

```diff
@@ -24,25 +24,83 @@ npm install @azure/arm-mediaservices
 npm install @azure/ms-rest-nodeauth@"^3.0.0"
 ```
 
+##### Create a .env file to use with the sample code
```

Comment:
We tend to keep the sample in the readme simpler with least amount of set up. While I would encourage the use of a .env file in the stand alone sample repo you have for media services, I would not recommend the same here. You can go with one of two options:
- Add a line saying that this sample expects the below environment variables to be updated as described, then add the below section 
- Or update the code snippet to have something like `const clientId = "add the clientId from the portal here"`

Reply:
I was wondering about that... since it would be tough to get it started without a link to a repo with a full sample.
Let me modify it a bit more for here. 
But once i get the samples up and running, i would link to those as well.